tests/Meilisearch.Tests/ObjectExtensionsTests.cs

```diff
@@ -10,8 +10,43 @@
 
 namespace Meilisearch.Tests
 {
+    public class FakeQuery
+    {
+        public DateTime? FakeDate { get; set; }
+        public string FakeString { get; set; }
+        public int? FakeInteger { get; set; }
+        public List<string> FakeStringList { get; set; }
+    }
+
```

Comment:
Why call every variable `Fake` instead of `mock` or `test` for example?

Reply:
It was just a preference, but the main reason was to avoid this `public string String { get; set; }`, which will "conflict" with the class name `String`. So to make all of them look the same, I opted to duplicate the class name.

Also, `Mock` is not good because those fields are not mocks at all :)
And `Test` could be a good choice! But again just preference 👍